lib/util/package.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
6 months after the initial discovery code was written, this is worthy of 100 <3s. Thank you.